db/migrate/20220722155443_add_cascade_to_user_associated_records.rb

```diff
@@ -0,0 +1,8 @@
+class AddCascadeToUserAssociatedRecords < ActiveRecord::Migration[6.1]
+  def change
+    remove_foreign_key :businesses, :users
+    add_foreign_key :businesses, :users, on_delete: :cascade
```

Comment:
We might not need this either if you get rid of the restrict_with_error on all models.